[EDU-6150] [EDU-6037] refactor: add new graphs info - Real-Time Metrics reference (#1535)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx

- **Total**: the sum of the requests according to the Status and Upstream Status.

By analyzing this graph, you can detect trends in request handling, pinpoint sources of errors, and optimize your infrastructure for better performance and reliability.

The graph includes the 10 most frequent request cases, but you can apply filters to access more specific information.

#### Bandwidth Saving

import BandwidthSaving from "~/includes/docs_help_center/en/real-time-metrics/edge-applications/bandwidth-saving/bandwidth-saving/index.md"

<BandwidthSaving />

#### Requests Breakdown

###### IP Address Information

The **IP Address Information** table provides an overview of the distribution of requests based on different geographical and network attributes, breaking them down by:

- **Remote Address**: individual IP addresses making requests.  
- **ASN**: Autonomous System Number, the network operator or organization responsible for the IP address.
- **Country**: the specific country from which the requests are coming.

> - **Region**: the geographic location where the requests are originating.

Reviewer: Sugestão: The geographic location from which the requests originate.

> - **Region**: the geographic location where the requests are originating.

Author: ```suggestion
- **Region**: the geographic location from which the requests originate.
```

The Bot Traffic graph shows the evolution of bot traffic over time. It presents historical data by identifying the traffic into:

- **Legitimate**: the request wasn't identified as an attack and there's enough data to ensure it isn't an attack, being considered legitimate human users.
- **Bad Bot**: the request reached the score threshold or it was identified as an attack.
- **Good Bot**: the request wasn't identified as an attack and matched any of the commonly used good bots, such as search engines.
- **Under Evaluation**: the request wasn't identified as a bot, but there isn't enough data to ensure it isn't an attack, being considered suspicious access.

It helps to identify periods of suspicious activity and allows you to detect patterns and anomalies. By hovering over the lines in the graph, a card will appear with more detailed information about the date, time, and the number of bots in each category.

###### Top Bot Traffic

The **Top Bot Traffic** graph is a pie chart that displays the distribution of bot traffic in percentages over the selected period. It categorizes traffic into:  

> - **Legitimate**: the request wasn't identified as an attack and there's enough data to ensure it isn't an attack, being considered legitimate legitimate human users.

Reviewer: ```suggestion
- **Legitimate**: the request wasn't identified as an attack and there's enough data to ensure it isn't an attack, being considered legitimate human users.
```
